static/app/components/nav/index.tsx

```diff
@@ -20,6 +20,7 @@ const NavContainer = styled('div')`
   position: sticky;
   top: 0;
   z-index: ${p => p.theme.zIndex.sidebarPanel};
+  user-select: none;
```

Comment:
Perhaps a controversial change from an accessibility standpoint, but in my opinion users who select text in the sidebar will always be doing so by mistake. Navigation labels are not "content" and the ability to select the text in this area of the app is confusing more than anything.

Reply:
Does the selection happen sometimes if folks click on the items (iirc double click can cause this)? I agree its annoying when it happens though, so I'm ok either way and I'm not sure there is a strong upside or downside here regardless if we allow this or not :)

Reply:
Right, accidental double clicks will select the label text. Always feels like a weird interaction to me on app navigation elements, but not a strongly held opinion.

Will move forward with this, but happy to remove if anyone objects in the future.